runtime/compiler/control/J9Options.cpp

```diff
@@ -2072,12 +2084,18 @@ J9::Options::setupJITServerOptions()
       {
       TR::PersistentInfo *persistentInfo = compInfo->getPersistentInfo();
       if (persistentInfo->getRemoteCompilationMode() == JITServer::SERVER)
+         {
          TR_VerboseLog::writeLineLocked(TR_Vlog_JITServer, "JITServer Server Mode. Port: %d. Connection Timeout %ums",
                persistentInfo->getJITServerPort(), persistentInfo->getSocketTimeout());
+         }
       else if (persistentInfo->getRemoteCompilationMode() == JITServer::CLIENT)
+         {
          TR_VerboseLog::writeLineLocked(TR_Vlog_JITServer, "JITServer Client Mode. Server address: %s port: %d. Connection Timeout %ums",
                persistentInfo->getJITServerAddress().c_str(), persistentInfo->getJITServerPort(),
                persistentInfo->getSocketTimeout());
+         TR_VerboseLog::writeLineLocked(TR_Vlog_JITServer,"Identifier for current client JVM: %PRIu64\n", 
```

Comment:
No, that's not how `PRIu64` is intended to be used; the format string should be expressed as:
```
  "Identifier for current client JVM: %" PRIu64 "\n"
```
`PRIu64` might expand to `"llu"` yielding a (more portable expression) version of what you had before.
Take note of the (required) spaces either side of `PRIu64`.
This pattern should be applied to all `printf` format strings you're adding or changing in this PR.

Reply:
Ohh thanks. There are three `llu` in this PR, let me fix those.